server.js

```diff
@@ -0,0 +1,73 @@
+var express = require('express');
+
+import { request, abort, } from 'request-promise';
+
+const port = process.env.PORT || 1337;
+
+const app = express();
+
+async function incoming2outgoingPayload(payload) {
```

Comment:
Do we need the `async` keyword here? I don't see any asynchronous functions being called.